Makefile

```diff
@@ -198,6 +199,18 @@ terraform/format: | guard/program/terraform
 	terraform fmt -recursive
 	@ echo "[$@]: Successfully formatted terraform files!"
 
+hcl/%: FIND_HCL := find . $(FIND_EXCLUDES) -type f \( -name '*.tfvars' -o -name "*.tf" -o -name "*.hcl" \)
```

Comment:
We have the terraform/% targets to handle .tf and .tfvars, and they're cleaner for that use case since they can just use -recursive. No need to go over the files again with this target

Reply:
ack, updated